addons/website/models/ir_ui_view.py

```diff
@@ -298,14 +298,19 @@ def filter_duplicate(self):
         if not current_website_id:
             return self.filtered(lambda view: not view.website_id)
 
+        specific_views_keys = set()
```

Comment:
This commit also still not respect the 72 characters rule for the commit message